packages/@webex/plugin-authorization-browser-first-party/src/authorization.js

```diff
@@ -240,6 +253,147 @@ const Authorization = WebexPlugin.extend({
       });
   },
 
+  /**
+   * Get an OAuth Login URL for QRCode. Generate QR code based on the returned URL.
+   * @instance
+   * @memberof AuthorizationBrowserFirstParty
+   * @throws {Error} When the request fails
+   * @returns {Promise<{verification_uri_complete: string, verification_uri: string, user_code: string, device_code: string, interval: number, expires_in: number}>}
+   */
+  initQRCodeLogin() {
+    return this.webex
+      .request({
+        method: 'POST',
+        service: 'oauth-helper',
+        resource: '/actions/device/authorize',
+        form: {
+          client_id: this.config.client_id,
+          scope: this.config.scope,
+        },
+        auth: {
+          user: this.config.client_id,
+          pass: this.config.client_secret,
+          sendImmediately: true,
+        },
+      })
+      .then((res) => {
+        const {user_code, verification_uri, verification_uri_complete} = res.body;
+        this.eventEmitter.emit('qrcode-login', {
+          eventType: 'user-code',
+          userData: {
+            user_code,
+            verification_uri,
+            verification_uri_complete,
```

Comment:
issue: in the JS code we use camelCase instead of snakeCase, please re-map the values from the server response to camelCase

```suggestion
            userCode: user_code,
            verificationUri: verification_uri,
            verificationUriComplete: verification_uri_complete,
```